va/va.h

```diff
@@ -1413,6 +1438,51 @@ typedef struct _VAEncMiscParameterAIR
     unsigned int air_auto; /* if set to 1 then hardware auto-tune the AIR threshold */
 } VAEncMiscParameterAIR;
 
+/*
+ * \brief Rolling intra refresh data structure for encoding.
+ */
+typedef struct _VAEncMiscParameterRIR
+{
+    union
+    {
+        struct
+	/**
+	 * \brief Indicate if intra refresh is enabled in column/row.
+	 *
+	 * App should query VAConfigAttribEncIntraRefresh to confirm RIR support
+	 * by the driver before sending this structure. The following RIR restrictions
+	 * apply:
+	 *  - No field encoding.
+	 *  - No B frames.
+	 *  - No multiple references.
```

Comment:
is the above a driver limitation only or the requirement for RIR? If it is a driver limitation,  please remove the above comments and add a mechanism to query the driver capabilities.